src/components/QuantumVolumeChart.js

```diff
@@ -984,7 +986,7 @@ function QuantumVolumeChart (props) {
       .style('visibility', 'hidden')
       .style('font-size', `${smallLabelSize}px`)
       .style('font-family', fontType)
-      .text((i) => d3.format('.2s')(y(i)))
+      .text((i) => d3.format('.2f')(y(i)))
```

Comment:
we can do the integer trick here as well

Reply:
Done in 0a9dfa7e8badccb8449fae3a974f114c2d91cf7a